src/main/generated/io/vertx/core/file/CopyOptionsConverter.java

```diff
@@ -27,16 +28,16 @@
 
   public static void fromJson(JsonObject json, CopyOptions obj) {
     if (json.getValue("atomicMove") instanceof Boolean) {
-      obj.setAtomicMove((Boolean) json.getValue("atomicMove"));
+      obj.setAtomicMove((Boolean)json.getValue("atomicMove"));
```

Comment:
this should be rather done in a separate PR that changes the _.templ_ that generates json conversion